buffer_space:
  bookend: 1
  bookstart: 1
  capacity: 32
  final_time: 0
default_starting_position:
  x: 0.0
  x_vel: 0.0
  y: 0.0
  y_vel: 0.0
  z: 0.0
  z_vel: 0.0
delta_tiers:
- 1
initial_uncertainty:
  known_starting_position:
    x: 0.0
    x_vel: 0.0
    y: 0.0
    y_vel: 0.0
    z: 0.0
    z_vel: 0.0
  ownship:
    x: 1.0
    x_vel: 0.2
    y: 1.0
    y_vel: 0.2
    z: 1.0
    z_vel: 0.2
  unknown_starting_position:
    x: 1000
    x_vel: 1.0
    y: 1000
    y_vel: 1.0
    z: 8
    z_vel: 1.0
measurement_topics:
  add_depth_bias: false
  depth: baro
  dvl: dvl
  imu_ci: strapdown/intersection
  imu_est: strapdown/estimate
  sonar: sonar_processing/target_list/associated
process_noise:
  blueteam:
    x: 0.0
    x_vel: 0.0
    y: 0.0
    y_vel: 0.0
    z: 0.0
    z_vel: 0.0
  ownship:
    x: 0.001
    x_vel: 0.001
    y: 0.001
    y_vel: 0.001
    z: 0.01
    z_vel: 0.1
  redteam:
    x: 0.1
    x_vel: 0.05
    y: 0.1
    y_vel: 0.05
    z: 0.1
    z_vel: 0.05
update_rate: 2
use_control_input: false
